Sources/Core/AWSClientRuntime/Sources/AWSClientRuntime/UserAgent/BusinessMetrics.swift

```diff
@@ -21,12 +27,20 @@ struct BusinessMetrics {
     ) {
         setFlagsIntoContext(config: config, context: context, headers: headers)
         self.features = context.businessMetrics
+        self.credentialFeatureIDs = context.selectedAuthScheme?.identity?.properties.get(
+            key: AttributeKey<[String]>(name: "CredentialFeatureIDs")
+        ) ?? []
+        self.manuallyConfiguredCredentialFeatureIDs = context.get(
+            key: AttributeKey<[String]>(name: "ManuallyConfiguredCredentialFeatureIDs")
+        ) ?? []
     }
 }
 
 extension BusinessMetrics: CustomStringConvertible {
     var description: String {
-        var commaSeparatedMetricValues = features.values.sorted().joined(separator: ",")
+        var commaSeparatedMetricValues = (
+            features.values.sorted() + credentialFeatureIDs + manuallyConfiguredCredentialFeatureIDs
+        ).joined(separator: ",")
```

Comment:
`credentialFeatureIDs` and `manuallyConfiguredCredentialFeatureIDs` will never be duplicate or contain duplicates. Whenever `manuallyConfiguredCredentialFeatureIDs` is non-empty, it means selected auth scheme is `smithy.api#noAuth` and therefore `credentialFeatureIDs` is empty. I.e., they're mutually exclusive in that if one is empty the other isn't, and vice versa.